### ISSUE TITLE
Project template updates unusable

### ISSUE THREAD

**Issue by bjones1** [RunestoneInteractive/RunestoneComponents#641]
I recently tested out the new projects templates available in `runestone/common/project_template` and see several issues which are illustrated in the image below.

![image](https://user-images.githubusercontent.com/2125584/43115937-1a54a552-8ecb-11e8-937a-c549c9540a14.png)

1. The navbar should probably collapse, but it doesn't for me. Therefore, it hides important text under it.
2. The navigation aids display "null" and "null" for me.

Any help would be appreciated. I'm building with current master.

**Comment by bnmnetp** [comment]
I have committed the cleaned up version of layout.html that we used in the fopp project. (648e246b4ffc47bc451ecf4bae6795024eb153d9)  It lacks the chapter progress bars, but I think they need a lot of cleanup to get it working and responsive.

There is a limited amount of real estate in that nav bar and I think we are trying to cram too much stuff in there.  I'd love to figure out a new/better template design that would make navigation and progress views better, but I think that is going to involve going back to a sidebar style template.
